documentation/README.md

```diff
@@ -59,6 +59,7 @@ The folder contains collection of docs and references for MSBuild, detailed info
 * [Target maps](wiki/Target-Maps.md)
 * [Managing parallelism in MSBuild](specs/resource-management.md)
 * [SDK resolution](specs/sdk-resolvers-algorithm.md)
+* [Nodes orchestration](wiki/Nodes-Orchestration.md)
```

Comment:
nit: I would suggest moving up a bit, to pair it with `[Low priority nodes](specs/low-priority-switch.md)` or even introduce parent `Nodes` and having 2 children under it.

Reply:
Good point - reordered